charts/member-agent-arc/values.yaml

```diff
@@ -14,6 +14,13 @@ refreshtoken:
   repository: mcr.microsoft.com/aks/fleet/refresh-token
   tag: "${REFRESH_TOKEN_IMAGE_VERSION}"
 
+crdInstaller:
```

Comment:
nit, none of the other values have upper case, and none of the other ones have a child `image:` either